### DISCUSSION TITLE
Hackathon: 2024-07-09

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#755] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by itcarroll** [comment]
@mfisher87 and I were the whole show for the first half, so we dug in to three quick PRs.

1. Reviewed and merged #613 
2. Reviewed, polished, and merged #744
3. Reviewed and merged #738

We were also thinking more about #749, which concerns random sampling within the integration tests.
- the tests that rely on random sampling should be changed to a fixed suite of collections
- Matt is proposing a list of concept_ids for each provider, using the cmr usage_score parameter to choose popular collections
- rely on earthaccess community, rather than random sampling, to surface collections that break earthaccess

**Comment by mfisher87** [comment]
We merged another one! https://github.com/nsidc/earthaccess/pull/584

Awesome work by @danielfromearth on this. He deserves a medal! :medal_military:
